javaclient/src/main/java/confluo/readme.md

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
I think it would be better to move this readme to the `confluo/streaming/` directory, since it does not apply to the entire `confluo/` directory. Also, it would make the readme more readable with the following changes: 
(1) There should be a heading; maybe `Confluo Pub-sub`? 
(2) Use sentence case instead of using lower-case for the entire document.